internal/ycsb/ycsb.go

```diff
@@ -53,6 +73,7 @@ func Run(ctx context.Context, dir string, args []string) (*config.TestResults, e
 
 	log.Printf("Running %s", strings.Join(cmd.Args, " "))
 
+	// FIXME ?
```

Comment:
@AlekSi should we also use the results of `loads` for metrics? (there are the `INSERT` operations measured).